enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -90,6 +90,7 @@ var (
 
 	forceRemoteSnapshotting = flag.Bool("debug_force_remote_snapshots", false, "When remote snapshotting is enabled, force remote snapshotting even for tasks which otherwise wouldn't support it.")
 	disableWorkspaceSync    = flag.Bool("debug_disable_firecracker_workspace_sync", false, "Do not sync the action workspace to the guest, instead using the existing workspace from the VM snapshot.")
+	debugDisableCgroup      = flag.Bool("debug_disable_cgroup", false, "Disable firecracker cgroup setup.")
```

Comment:
Did you mean to add this flag to vmstart?

Reply:
(I think you meant set the flag to true in vmstart?) done :+1:

Reply:
Did you mean to set it to true?

Also could you merge this? It would make development on my GCP VM smoother

Reply:
oops, fixed